src/test/java/group1/mips_simulator/components/ExecutionInstructionsTest.java

```diff
@@ -0,0 +1,96 @@
+package group1.mips_simulator.components;
+
+import group1.mips_simulator.components.cpuParts.ConditionCode;
+import group1.mips_simulator.components.cpuParts.RegisterFile;
+import group1.mips_simulator.components.instructionParts.Field;
+import group1.mips_simulator.components.instructionParts.OpCode;
+import group1.mips_simulator.components.instructionParts.RXIA_Instruction;
+import group1.mips_simulator.components.memParts.Memory;
+import org.junit.jupiter.api.Test;
+
+import java.util.Vector;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ExecutionInstructionsTest {
+
+    Computer buildComputer() {
```

Comment:
will be taking inspiration for this to write unit tests tonight, cool stuff

Reply:
🧪